pycon/settings.py

```diff
@@ -469,6 +469,7 @@ def _(x):
     "ep2019",
     "ep2020",
 )
+EARLY_BIRD_ORDER_LIMIT = config("EARLY_BIRD_ORDER_LIMIT", default=210, cast=int)
```

Comment:
I'd raise the default to 220, since experience shows that quite a few people leave tickets in the cart without actually ordering. The limit doesn't have to be a hard limit, only prevent going way beyond the limit.